### DISCUSSION TITLE
Games.

### DISCUSSION THREAD

**Discussion by ItzEclipser** [skyline-emu/skyline#157] [Q&A]
This might come off as impatient but that isn't my intention. I was just wondering if this emulator can run any games as of now?

**Comment by harry2001a** [comment]
I think there are one or two games that take you to the title screen. Otherwise...no no games yet

**Reply by ItzEclipser** [reply]
Which ones are they?

**Reply by harry2001a** [reply]
I think Bastion is one of them. There is a YouTube video that shows it. It only gives you a picture with the title of the game and nothing else
